bin/oca_checklog_odoo

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+#
+# Check if odoo logs contain errors. Assumes logs will come from stdin
+#
+
+if [ -n "${OCA_ENABLE_CHECKLOG_ODOO}" ]; then
+    checklog-odoo
+else
+    checklog-odoo -i ".*"
```

Comment:
I know I proposed this approach, but I wonder if we could use a simple unix command instead such as `cat`?

Reply:
I mean, specifically for the case when we don't want to check logs

Reply:
Yeah, that would probably make more sense. Changed it just now